.travis.yml

```diff
@@ -1,46 +1,38 @@
-# The language in this case has no bearing - we are going to be making use of "conda" for a
-# python distribution for the scientific python stack.
-language: c
+language: generic
 
 os:
-# - linux
  - osx
 
 sudo: false
 
 env:
-    global:
-        # Add the BINSTAR_TOKEN environment variable. This has been generated with "anaconda auth --create --name conda-recipes-scitools-2016 -o SciTools"
-        # then "travis encrypt BINSTAR_TOKEN=<token_id> --repo SciTools/conda-recipes-scitools"
-        secure: "gUnBiclSuzSUSyJCVM2acIjry1VoqDlqWCkIDYx4VHJfRn7VXrav+G0cn2nYW7L8ch3/tnLqXLNEYRoBq8tbNOoOnnksqd0zTDNuUD4HkaQ47fNTsSbBVDzgDSiTvj9hjUFM7hQ0ju1BffG/uu43gl1mOaSh+R5/jdJ+iGlF+gg="
-
-addons:
-    apt:
-        packages:
-            # Install makeinfo [sic] - needed for constructing configuration
-            # files via autoreconf (in particular for udunits)
-            - texinfo
-
-            # Unit testing framework for C (used by libmo_unpack)
-            - check
+  global:
+    secure: "gUnBiclSuzSUSyJCVM2acIjry1VoqDlqWCkIDYx4VHJfRn7VXrav+G0cn2nYW7L8ch3/tnLqXLNEYRoBq8tbNOoOnnksqd0zTDNuUD4HkaQ47fNTsSbBVDzgDSiTvj9hjUFM7hQ0ju1BffG/uu43gl1mOaSh+R5/jdJ+iGlF+gg="
 
 before_install:
-    - brew update && brew install check
+  - brew remove --ignore-dependencies --force $(brew list)
 
 install:
-    # Set the numpy variable. This isn't used, but conda-build complains if we haven't set it already.
-    - export CONDA_NPY=19
-    # Set the name of the channel label to upload to.
-    - export LABEL_NAME=main
-    # Only run the upload if the binstar token is present.
-    - export UPLOAD_CHANNELS=$(if [ -z ${BINSTAR_TOKEN+x} ]; then echo ""; else echo "--upload-channels scitools/label/${LABEL_NAME}"; fi)
-    - curl -L -O https://raw.githubusercontent.com/conda-forge/conda-smithy/master/bootstrap-obvious-ci-and-miniconda.py
-    - python bootstrap-obvious-ci-and-miniconda.py ~/miniconda x64 3 --without-obvci && source ~/miniconda/bin/activate root
-    - conda config --add channels scitools
-    - conda config --set show_channel_urls True
-    - conda install --yes --quiet -c conda-forge conda-build-all conda-build==2.0.10
-    - conda install -n root --yes --quiet conda==4.2.13 jinja2 anaconda-client
-    - conda list
+  - URL="https://repo.continuum.io/miniconda/Miniconda3-latest-MacOSX-x86_64.sh"
+  - curl -L $URL -o miniconda.sh
+  - bash miniconda.sh -b -p $HOME/miniconda
+  - export PATH="$HOME/miniconda/bin:$PATH"
+  - hash -r
+  - conda update --yes conda
+
+  # Configure channels.
+  - conda config --set show_channel_urls True
+  - conda config --add channels scitools
+
+  - conda install --yes -c conda-forge conda-build-all
+  # https://github.com/SciTools/conda-build-all/issues/79
+  - conda install --yes conda==4.2.16
```

Comment:
We cannot use `conda >=4.3` with `conda-build-all`. See https://github.com/SciTools/conda-build-all/issues/79